### ISSUE TITLE
Docs: React snippets lack contrast

### ISSUE THREAD

**Issue by daniel-ac-martin** [daniel-ac-martin/NotGovUK#35]
@karypun points out that the new syntax highlighting on the React snippets might be lacking in contrast and so pose a risk to accessibility.

If possible, it would also be nice to bring it into line with the HTML colours anyway. Allegedly that is using a 'GitHub' theme so that would be a good place to start.

**Comment by daniel-ac-martin** [comment]
GitHub theme candidates:
- https://github.com/PrismJS/prism-themes/blob/master/themes/prism-ghcolors.css
- https://github.com/LukeAskew/prism-github
- https://github.com/ElForastero/prismjs-github

**Comment by daniel-ac-martin** [comment]
Theme generator: http://k88hudson.github.io/syntax-highlighting-theme-generator/www/

**Comment by daniel-ac-martin** [comment]
Went with `prismjs-github`.

Unless this is raised as a specific problem in an accessibility assessment, we can probably close this.
